main.py

```diff
@@ -0,0 +1,183 @@
+from fastapi import (
+    FastAPI,
+    Depends,
+    HTTPException,
+    status,
+    security
+)
+from sqlalchemy.orm import Session
+
+import crud, schemas, utils
+
+
+app = FastAPI()
+
+
+@app.get("/")
+def root():
+    return {"Hello": "World"}
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(
+    skip: int = 0,
+    limit: int = 10,
+    author_name: str | None = None,
+    sort_field: str | None = None,
+    db: Session = Depends(utils.get_db)
+):
+    return crud.get_author_list(
+        db, skip=skip,
+        limit=limit,
+        author_name=author_name,
+        sort_field=sort_field,
+    )
+
+
+@app.get("/authors/{author_id}", response_model=schemas.Author)
```

Comment:
I don't think it's crucial, but as a rule endpoints should end with /, fix this everywhere

Reply:
thanks, will do